integration-cli/docker_cli_update_unix_test.go

```diff
@@ -187,3 +176,25 @@ func (s *DockerSuite) TestUpdateStats(c *check.C) {
 	c.Assert(preMemLimit, checker.Equals, curMemLimit)
 
 }
+
+func (s *DockerSuite) TestUpdateRestartPolicy(c *check.C) {
```

Comment:
This should be moved out of the `_unix_test` file since it is expected to work on Windows.